src/pages/propose.astro

```diff
@@ -1,10 +1,151 @@
+  // Just check if it contains '@'
+  return !!email && email.includes('@');
+}
 ---
 
-<Layout title="Tulsa Web Devs">
-  <main class="grid w-screen h-screen bg-white text-4xl place-items-center">
-    Proposal Features Here!
+<Layout title="propose">
+  <main>
+    <img
+      alt="Tulsa Web Devs logo"
+      src="/twd-logo.svg"
+      height="auto"
+      width="100%"
+    />
+
+    <div class="flex flex-col items-center m-2" id="header">
+      <h1 class="text-3xl max-md:text-2xl">Propose an idea!</h1>
+      <h2 class="text-2xl max-md:text-xl">
+        Have an idea?! Submit it! Just fill out this form!
+      </h2>
+    </div>
+
+    <form method="POST" id="proposalForm" class="m-4">
+      <div class="p-4 flex flex-col">
+        <div id="name-fields" class="shrink p-2">
+          <label for="name" class="text-l font-semibold">Name:</label>
+          <input
+            type="text"
+            name="name"
+            required
+            minlength={6}
+            placeholder="Enter your name, nickname, or alias..."
+            class="rounded-small w-3/4 max-md:w-full"
+          />
+        </div>
+        <div id="email-field" class="shrink p-2">
+          <label for="email" class="text-l font-semibold">Email:</label>
+          <input
+            type="email"
+            name="email"
+            placeholder="myemailaddress@example.com..."
+            class="rounded-small w-3/4 max-md:w-full"
+          />
+        </div>
+      </div>
+
+      <div class="p-4" id="topic-summary-fields">
+        <label for="topic" class="text-l font-semibold">Topic:</label>
+        <textarea
+          id="topic"
+          name="topic"
+          required
+          minlength={6}
+          class="rounded-sm w-full"
+          placeholder="Topic title, framework name, main subject..."></textarea>
+        <label for="summary" class="text-l font-semibold">Summary:</label>
+        <textarea
+          id="summary"
+          name="summary"
+          required
+          minlength={6}
+          rows={5}
+          class="rounded-sm w-full"
+          placeholder="Tell us more about it here..."></textarea>
+      </div>
+
+      <div
+        class="checkboxes border-t-4 border-b-4 flex items-center justify-start text-left m-5 p-4 gap-5"
+        id="checkboxes"
+      >
+        <input
+          type="checkbox"
+          id="project"
+          name="proposal-type"
+          value="project"
+        />
+        <label for="project" class="text-l font-semibold">
+          {' '}
+          Project!{' '}
+        </label>
+        <input type="checkbox" id="talk" name="proposal-type" value="talk" />
+        <label for="talk" class="text-l font-semibold">
+          {' '}
+          Talk!{' '}
+        </label>
+        <input type="checkbox" id="lab" name="proposal-type" value="lab" />
+        <label for="lab" class="text-l font-semibold">
+          {' '}
+          Lab!
+        </label>
+      </div>
```

Comment:
issue: We're using checkboxes where the options are mutually exclusive
suggestion: Use radio buttons instead

issue: The field names for the options don't match the [api spec](tulsawebdevs.org/docs/api)
suggestion: Rename the option fields from `proposal-type` to `type`

issue: The options here don't match the [api spec](tulsawebdevs.org/docs/api)
suggestion: Refactor these values to match the enum members expected by the `/proposals` endpoint for the `type` field

Reply:
Happy to help!

The check boxes are now radio buttons. The names are now "type".

![image](https://github.com/tulsawebdevs/website/assets/98660173/7e395a81-aca3-4850-90c3-82f8bf46171e)